monai/networks/layers/utils.py

```diff
@@ -92,3 +92,26 @@ def get_dropout_layer(name: Union[Tuple, str, float, int], dropout_dim: Optional
         drop_name, drop_args = split_args(name)
     drop_type = Dropout[drop_name, dropout_dim]
     return drop_type(**drop_args)
+
+
+def get_pool_layer(name: Union[Tuple, str], spatial_dims: Optional[int] = 1):
+    """
+    Create a pooling layer instance.
+
+    For example, to create normalization layers:
```

Comment:
Docstring needs an update here

Reply:
sorry for the typo, updated in the latest commit.
Thanks.